### PR TITLE
Fix SerDe Issue 195

### PR DESCRIPTION
**Pull request by leerho**

### CODE REVIEW

File: src/test/java/com/yahoo/sketches/quantiles/DoublesUnionImplTest.java [hunk 1]
```diff
@@ -693,6 +693,15 @@ public void isSameResourceDirect() {
     Assert.assertFalse(union.isSameResource(mem2));
   }
 
+  @SuppressWarnings("unused")
+  @Test
+  public void checkSerDeIssue165() {
```
Comment (hpx7) [inline src/test/java/com/yahoo/sketches/quantiles/DoublesUnionImplTest.java]: did you mean 195?

**Comment by AlexanderSaydakov** [comment]
The fix looks fine to me. Let's rename the test to either checkSerDeIssue195 or, perhaps, something like emptyUnionSerDe and assert a few things about the result.

**Comment by hpx7** [comment]
Thanks for the quick turnaround! Could we tag a release for this?

**Comment by leerho** [comment]
Hopefully soon.  We need a few days to review all the changes we have made
since the last release.

**Pull request merged**